features/cli.feature

```diff
@@ -18,18 +18,7 @@ Feature: CLI
     And the output should match /Commands:/
 
   Scenario: When overriding a setting from the config file on the command line
-    Given a puppet module "puppet-test" from "fakenamespace"
-    And a file named "managed_modules.yml" with:
-      """
-      ---
-        - puppet-test
-      """
-    And a file named "modulesync.yml" with:
-      """
-      ---
-      namespace: default
-      """
-    And a git_base option appended to "modulesync.yml" for local tests
+    Given a basic setup with a puppet module "puppet-test" from "fakenamespace"
```

Comment:
Should this be `from "default"` to keep the test correct?

Reply:
The test is `When overriding a setting from the config file on the command line` so it keeps the test correct.

Reply:
Oups, you're right!